R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -71,7 +71,25 @@ test_that("glm and predict", {
            data = iris, family = poisson(link = identity)), iris))
   expect_true(all(abs(rVals - vals) < 1e-6), rVals - vals)
 
-  # Test stats::predict is working
+  # Test model save/load
+  modelPath <- tempfile(pattern = "GLM", fileext = ".tmp")
+  ml.save(model, modelPath)
+  expect_error(ml.save(model, modelPath))
+  ml.save(model, modelPath, overwrite = TRUE)
+  m2 <- ml.load(modelPath)
+  s2 <- summary(m2)
+  expect_equal(s$rCoefficients, s2$rCoefficients)
```

Comment:
I agree. Should we do this test for all Gaussian, Poisson and Binomial family. I am assuming doing for either one of them should be sufficient?